packages/camera/tizen/src/camera_device.cc

```diff
@@ -852,6 +882,27 @@ void CameraDevice::UpdateStates() {
   GetRecorderState(recorder_state_);
 }
 
+bool CameraDevice::RegistAvailableMediaPacket(media_packet_h packet) {
+  std::lock_guard<std::mutex> lock(mutex_);
+  if (packet_) {
```

Comment:
It's not much of a difference, but how about replacing it with a new packet?

Reply:
please review again, I updated this PR